src/_component/connect/connect.scss

```diff
@@ -83,7 +90,7 @@
   [class^="icon__"] {
     width: 100%;
     height: 100%;
-    font-size: 10vw;
+    font-size: 8vw;
```

Comment:
I wouldn't use viewport for font-size directly, read up this article on Fluid Typography